packages/xo-server/src/api/vm.js

```diff
@@ -756,6 +757,30 @@ export { convertToTemplate as convert }
 
 // -------------------------------------------------------------------
 
+export async function cloneToTemplate({ vm }) {
+  if (vm.power_state === 'Running') {
+    throw incorrectState({
+      actual: 'Running',
+      expected: 'Halted',
+      object: vm.$id,
+      property: 'power_state',
+    })
+  }
```

Comment:
Why is this necessary?

Reply:
Like it said in the issue, the user want to create a template from a snapshot or from a VM. This verification is in the case of we try to create a template from a VM.
I read in the `Xapi` documentation, to clone a VM, this one have to be `halted`

Reply:
You don't call `VM.clone`, you call `Xapi#copyVm`, IMHO, if the test is needed it should be migrated there.